### PR TITLE
update MariaDB manually (bsc#1132852, SOC-9022)

### PR DESCRIPTION
**Pull request by csymons-suse**

### CODE REVIEW

**Comment by asettle** [comment]
@csymons-suse - Rick will be out until the 3rd of June. Would you like me/Dima to review as well or do you have another SME for this?

**Comment by csymons-suse** [comment]
> @csymons-suse - Rick will be out until the 3rd of June. Would you like me/Dima to review as well or do you have another SME for this? Or, if it's not urgent, leave it for Rick?

Rick already reviewed https://github.com/SUSE-Cloud/doc-cloud/pull/971. 
`This looks good but we should add the same section to the Ardana docs too.`
So all that's needed is to have someone check that this has been done as Rick requested in bsc#1132852. 

He said that the content was needed for Cloud 7, 8, 9/Crowbar&CLM. He requested the location for Crowbar. There was no equivalent location for CLM; so I made it up.

Would be good to have you or Dima look at it. It's a high priority bug/Jira. 

The content on PR 971, 978, 979 is the same-ish.

File: xml/depl_maintenance.xml [hunk 1]
```diff
@@ -640,10 +640,10 @@ Error during prechecks. 4 Byte UTF8 characters found in the source database.</sc
        the target databases will be replaced with the latest one in the source databases.
        </para>
        <para>
-	After the test migration and before the actual migration, it is recommended to run the
-	<command>psql2mysql purge-tables</command> command to purge tables in
-	the target database. While this step is optional, it speeds up the
-	migration process.
+        After the test migration and before the actual migration, it is recommended to run the
```
Comment (asettle) [inline xml/depl_maintenance.xml]: we recommend

File: xml/depl_maintenance.xml [hunk 2]
```diff
@@ -711,27 +711,27 @@ migrate</screen>
...
+         Remove the packages on all cluster nodes:
+        </para>
+        <screen>&prompt.root;zypper rm postgresql94 postgresql94-server</screen>
        </step>
        <step>
-	<para>
-	 If you choose not to upgrade to &productname; 8 right away, delete unused pacemaker resource from one cluster node:
-	</para>
+        <para>
+         If you choose not to upgrade to &productname; 8 right away, delete unused pacemaker resource from one cluster node:
```
Comment (asettle) [inline xml/depl_maintenance.xml]: Could you wrap the text here? Can't see the end :)

File: xml/operations-maintenance-mariadb-manual-update.xml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+ type="text/xml"
+ title="Profiling step"?>
+<!DOCTYPE section [
+ <!ENTITY % entities SYSTEM "entity-decl.ent"> %entities;
+]>
+<section xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink" xml:id="mariadb-manual-update">
+ <title>Updating &mariadb; with Galera</title>
+   <para>
+   When using Pacemaker, updating &mariadb; with Galera must be done
+   manually. &crow; will not install updates automatically. In particular, this
```
Comment (asettle) [inline xml/operations-maintenance-mariadb-manual-update.xml]:
```suggestion
manually. &crow; does not install updates automatically. In particular, this
```

**Pull request merged**